### PR TITLE
update userguide

### PR DESCRIPTION
**Pull request by brendanng7**
Closes https://github.com/AY2425S1-CS2103T-F14b-3/tp/issues/87

### CODE REVIEW

File: docs/images/HelpWindow.png
Comment (Zenkoh1) [inline docs/images/HelpWindow.png:1]: I think the help window js got updated with other commands, can use that instead

**Review by Zenkoh1 — COMMENTED** [review]
also the history command was added in the summary table in `UserGuide.md` but wasn't added in the help window, I think can js help to copy paste it over

File: docs/images/HelpWindow.png
Comment (brendanng7) [reply docs/images/HelpWindow.png:1]: okk I was thinking of changing the UI.png too as its outdated but wasn't sure. what do you think?

Comment (Zenkoh1) [reply docs/images/HelpWindow.png:1]: sure why not

**Comment by brendanng7** [comment]
a lot of codecov is missing because the methods I added to the Commands are for getting the text values to put into the table in help window.

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 1]
```diff
@@ -98,4 +132,51 @@ private void copyUrl() {
...
+                new EditCommand.EditPersonDescriptor()));
+        commandList.add(new FindCommand(new NameContainsKeywordsPredicate(new ArrayList<>())));
+        commandList.add(new HistoryCommand(Index.fromOneBased(1)));
+        commandList.add(new ListCommand());
+        try {
+            commandList.add(new MarkCommand(Index.fromOneBased(1),
+                    new ContactRecord(ParserUtil.parseDate("2020-13-01"), "")));
+        } catch (ParseException e) {
+            logger.warning("HelpWindow parse error: Date format is incorrect.");
+        }
```
Comment (Zenkoh1) [inline src/main/java/seedu/address/ui/HelpWindow.java:178]:
```suggestion

        commandList.add(new MarkCommand(Index.fromOneBased(1),
                    new ContactRecord(LocalDate.parse("2020-13-01"), "")));

```
then don't need to have a try catch, cuz it's abit weird to have a try catch for a date that is hard coded

File: docs/images/HelpWindow.png
Comment (Zenkoh1) [inline docs/images/HelpWindow.png:1]: actually how does transparent background work on Markdown, is it displayed correctly

**Review by Zenkoh1 — APPROVED** [review]
otherwise LGTM

File: docs/images/HelpWindow.png
Comment (brendanng7) [reply docs/images/HelpWindow.png:1]: good question... we will find out soon. it should be displayed correctly just with more spacing between the words and the image

**Pull request merged**